codalab/server/bundle_manager.py

```diff
@@ -320,14 +320,8 @@ def _schedule_run_bundles_on_workers(self, workers, staged_bundles_to_run):
             reverse=True,
         )
 
-        # Get all the run_uuids from workers
-        run_uuids = []
-        for worker in workers.workers():
-            run_uuids.extend(worker["run_uuids"])
-        # Get the running bundles that exist in the bundle table
-        running_bundles = self._model.batch_get_bundles(uuid=run_uuids)
-        # Build a dictionary which maps from uuid to running bundle
-        uuid_to_running_bundles = {bundle.uuid: bundle for bundle in running_bundles}
+        # Build a dictionary which maps from uuid to running bundle and bundle_resources
+        running_bundles_cache = self._build_running_bundles_cache(workers)
```

Comment:
Not sure we need to call it a "cache", since we're not populating it lazily...we're just get information about running bundles.

Reply:
Renamed to `running_bundles_info`